### PR TITLE
Fix acceptance tests

### LINKED ISSUE #975: CI failing for some federated share tests
See PR #974 

https://drone.owncloud.com/owncloud/activity/1975/23/15
```
runsh: Total unexpected failed scenarios throughout the test run:
webUIActivitySharingExternal/sharingFederation.feature:31
webUIActivitySharingExternal/sharingFederation.feature:38
```

https://drone.owncloud.com/owncloud/activity/1975/24/15
```
runsh: Total unexpected failed scenarios throughout the test run:
webUIActivitySharingExternal/sharingFederation.feature:31
webUIActivitySharingExternal/sharingFederation.feature:38
```

```
  Scenario: Sharing a file/folder with a remote server should be listed in the activity list of a sharee eventhough they have not accepted the share # /var/www/owncloud/testrunner/apps/activity/tests/acceptance/features/webUIActivitySharingExternal/sharingFederation.feature:31
    Given user "Carol" from server "REMOTE" has shared "textfile0.txt" with user "Brian" from server "LOCAL"                                         # FederationContext::userFromServerHasSharedWithUserFromServer()
    And user "Carol" from server "REMOTE" has shared "simple-folder" with user "Brian" from server "LOCAL"                                           # FederationContext::userFromServerHasSharedWithUserFromServer()
    When the user browses to the activity page                                                                                                       # WebUIActivityContext::userHasBrowsedToTheActivityPage()
    Then the activity number 1 should contain message "You received a new remote share simple-folder from Carol@…" in the activity page              # WebUIActivityContext::theActivityNumberShouldContainMessageInTheActivityPage()
      Failed asserting that 'You received a new remote share from Carol@…' contains "You received a new remote share simple-folder from Carol@…".
    And the activity number 2 should contain message "You received a new remote share textfile0.txt from Carol@…" in the activity page               # WebUIActivityContext::theActivityNumberShouldContainMessageInTheActivityPage()
SCENARIO RESULT: (fail)

  Scenario: remote sharee does not get new activity message after accepting the pending share                                           # /var/www/owncloud/testrunner/apps/activity/tests/acceptance/features/webUIActivitySharingExternal/sharingFederation.feature:38
    Given user "Carol" from server "REMOTE" has shared "textfile0.txt" with user "Brian" from server "LOCAL"                            # FederationContext::userFromServerHasSharedWithUserFromServer()
    When user "Brian" from server "LOCAL" accepts the last pending share using the sharing API                                          # FederationContext::userFromServerAcceptsLastPendingShareUsingTheSharingAPI()
    And the user browses to the activity page                                                                                           # WebUIActivityContext::userHasBrowsedToTheActivityPage()
    Then the activity number 1 should contain message "You received a new remote share textfile0.txt from Carol@…" in the activity page # WebUIActivityContext::theActivityNumberShouldContainMessageInTheActivityPage()
      Failed asserting that 'You received a new remote share from Carol@…' contains "You received a new remote share textfile0.txt from Carol@…".
SCENARIO RESULT: (fail)
```

The file name is not in the activity message.

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/activity/issues/975 originated from https://github.com/owncloud/core/pull/38880 and https://github.com/owncloud/activity/pull/972

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -94,3 +94,35 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (phil-davis) [inline .htaccess:97]: Accidental commit of .htaccess

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiCapabilities-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/55/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioningGroups-v2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/62/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliExternalStorage-10.6.0-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/126/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIMoveFilesFolder-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/136/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/92/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingExt2-latest-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/162/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProxySmoke-8-6-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30797/175/1

**Comment by JanAckermann** [comment]
@phil-davis we added a new subjectparam that's why we ran in the == 2 check was not fulfilled and the wrong text was rendered

**Review by phil-davis — APPROVED** [review]
LGTM

**Review by JammingBen — CHANGES_REQUESTED** [review]
We should do the same in `translateShort()`: https://github.com/owncloud/core/blob/cc899608ee0c4c21da8d44f7a19d84db15eb8c33/apps/files_sharing/lib/Activity.php#L313

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiMain-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30800/58/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiTrashbin-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30800/88/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToRoot2-git-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30800/109/1

**Review by phil-davis — APPROVED** [review]

**Review by JammingBen — APPROVED** [review]

**Comment by phil-davis** [comment]
Activity CI passed last night :rocket:

**Pull request merged**